notifications/views/emailtemplate.py

```diff
@@ -122,6 +122,28 @@ def get_context_data(self, **kwargs):
         context = super(CycleEmailTemplateTest, self).get_context_data(**kwargs)
         context['template'] = get_object_or_404(CycleEmailTemplate,
                                                 id=self.kwargs['pk'])
+        company = Company.objects\
```

Comment:
We avoid using \ to keep code under 80 characters.
You can define it like:
    
    company = ( 
        Company.objects.filter(
           group=context['template'].group
         ).order_by('?').first()
    )